bridgesync/processor.go

```diff
@@ -485,13 +493,19 @@ func (p *processor) GetBridgesPaged(
 		pageSize = 1
 	}
 	offset := (pageNumber - 1) * pageSize
-	if offset >= uint32(count) {
-		p.log.Debugf("offset is larger than total bridges (page number=%d, page size=%d, total bridges=%d)",
-			pageNumber, pageSize, count)
-		return nil, 0, db.ErrNotFound
+	if offset >= uint32(bridgesCount) {
+		p.log.Debugf("offset is greater than total bridges (page number=%d, page size=%d, total bridges=%d)",
+			pageNumber, pageSize, bridgesCount)
+		return nil, 0, fmt.Errorf("provided offset is greater than the total bridges count (offset=%d, total count=%d)",
```

Comment:
I would just change the messages in these `offset >= count` errors:

```suggestion
		return nil, 0, fmt.Errorf("provided page number is invalid for given page size and total number of bridges (page number=%d, page size=%d, total count=%d)",
```

Reply:
Fixed https://github.com/agglayer/aggkit/pull/391/commits/c0cea262cc91f7adf7ea18708f48cef10fbf46fb